File: 18_spring_security/token/src/main/java/com/yologger/samples/token/controller/TestController.java
package com.yologger.samples.token.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/test")
public class TestController {

    @GetMapping
    @RequestMapping("/test1")
    String test1() {
        return "test1";
    }
}

File: 04_spring_kotlin/src/main/kotlin/com/yologger/samples/spring_kotlin/controller/TestController.kt
package com.yologger.samples.spring_kotlin.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/test")
class TestController {
    @GetMapping
    @RequestMapping("/test1")
    fun test1(): String {
        return "test1"
    }
}